src/generator/www/js/main.js

```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
+
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier '$url' does not match the pattern '^[a-z]+([A-Z][a-z]+)*$'.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710866493/issues/source?bid=3233857&fileBranchId=3344140#l5)